friends/src/App.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+
+class App extends Component {
+  state = {
+    lists:[],
```

Comment:
I'd change `lists` to `friends` just to avoid confusion, but there's an argument for keeping the name generic as well.